docs/sidebars.ts

```diff
@@ -58,7 +60,7 @@ const sidebars: SidebarsConfig = {
         },
         {
           type: "category",
-          label: "Lodestar Light Prover",
+          label: "Lodestar Prover",
```

Comment:
imo we shouldn't prefix this with "Lodestar" in our docs, we need to come up with a proper name for Prover, maybe "Execution Prover"?

Reply:
we  really need to remove the Lodestar prefixes, this looks really redundant

![image](https://github.com/ChainSafe/lodestar/assets/38436224/f0036bfc-8507-46e3-9ac7-a9c24186981e)

what other than the Lodestar light client would be in the Lodestar docs?

Reply:
> imo we shouldn't prefix this with "Lodestar" in our docs, we need to come up with a proper name for Prover, maybe "Execution Prover"?
In all of our content so far, such as Nazar's presentations, documentation, articles, etc. we've been addressing this as the "Lodestar Prover." I'd prefer that we stick to this name unless we have some more consensus from others about changing this to not confuse it with content that's already out there.

> this looks really redundant
We can probably fix this by asking "What does this page have to do with the Light Client / Prover?" It's basically the _Usage_ page of those tools. We should probably rename the link in Navigation to "Usage".

> what other than the Lodestar light client would be in the Lodestar docs?
Do you mean what other tools would go under "Developer Tools" section? If yes, when we develop other tooling that helps devs better utilize Lodestar for experimental purposes. An example in the future would probably be a page for the Flare CLI and other future ideas we haven't come up with yet.

Reply:
To add consistency, we should probably also rename the nav link "Lodestar Eth Consensus Lightclient Prover" to also just be "Usage" also.

Reply:
> Do you mean what other tools would go under "Developer Tools" section?

No, this was an ironic question, we can just say "Light client", we don't need to add Lodestar everywhere, we also don't do it for beacon node / validator client, Testing, etc. In theory we could prefix everything with `Lodestar` but that's obviously redundant since this are the Lodestar docs, so it's unlikely someone would assume if we just say "Light client" here that we talk about Nimbus light client, or any other Light client.

It just does not look clean and well thought through if we repeat ourselves like this.

Reply:
Fair point. We can just rename it to "Light Client" and "Prover" on the nav. Once these nav changes are solidified, I'll go deeper into the content and start standardizing terms for consistency.